### ISSUE TITLE
Iterating over dict items() breaks in Python 3

### ISSUE THREAD

**Issue by davisagli** [zopefoundation/RestrictedPython#129]
In restricted Python 3, code like this:

```
d = {'a': 'b'}
for k, v in d.items():
    pass
```

results in an Unauthorized exception:

```
The container has no security assertions.  Access to None of dict_items([('a', 'b')]) denied.
```

**Comment by loechel** [comment]
@davisagli that might be a combination issue in Zope with AccessControl and RestrictedPython.

Within RestrictedPython this is intended that looping over things need to be configured.
There is a paragraph in the documentation about that: https://restrictedpython.readthedocs.io/en/latest/usage/index.html#necessary-setup

in #130 I did add an test that shows how that might be used. But it did not solve the Issue in Zope Context.

**Comment by icemac** [comment]
Maybe fixed by zopefoundation/AccessControl#90.

@davisagli Did the error occur in pure RestrictedPython or was AccessControl involved?

**Comment by davisagli** [comment]
@icemac I probably found this while working on Plone, so yes, with AccessControl. It makes sense to me that zopefoundation/AccessControl#90 would fix this.
